src/realm/dictionary.hpp

```diff
@@ -77,21 +77,33 @@ class Dictionary final : public CollectionBaseImpl<CollectionBase> {
     std::pair<Iterator, bool> insert(Mixed key, Mixed value);
     std::pair<Iterator, bool> insert(Mixed key, const Obj& obj);
 
+    Obj create_and_insert_linked_object(Mixed key);
+
     // throws std::out_of_range if key is not found
     Mixed get(Mixed key) const;
     // Noexcept version
     util::Optional<Mixed> try_get(Mixed key) const noexcept;
     // adds entry if key is not found
     const Mixed operator[](Mixed key);
 
+    Obj get_object(StringData key)
+    {
+        auto val = try_get(key);
+        Obj obj;
+        if (val && (*val).is_type(type_Link)) {
+            return get_target_table()->get_object((*val).get<ObjKey>());
+        }
+        return obj;
+    }
+
     bool contains(Mixed key);
     Iterator find(Mixed key);
 
     void erase(Mixed key);
     void erase(Iterator it);
 
     void nullify(Mixed);
-    void remove_backlinks();
+    bool remove_backlinks(CascadeState& state) const;
```

Comment:
What does the return value mean here?

Reply:
All these functions that are about removing backlinks returns a boolean telling if there are any embedded children to remove.

Reply:
Can you put that in a comment? :)

Reply:
Hmm, turns out that it is not even used. Will remove.